src/index.js

```diff
@@ -1,7 +1,13 @@
+var completer = require('./completer');
 global.locusModules = {};
 global.locusModules.deasync = require.resolve('deasync');
 global.locusModules.color = require.resolve('cli-color');
 global.locusModules.stackTrace = require.resolve('stack-trace');
+global.locusReadLine = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  completer: completer
```

Comment:
I think you should add this right after line 24, I moved readline into the function before.